slack.json

```diff
@@ -26,7 +26,7 @@
     },
     {
       "github": "robertparkinson",
-      "slack": "robertp"
+      "slack": "robertparkinson"
```

Comment:
does he have a username? normally you want to use the slack member id

Reply:
im confused 🙈he is on slack now with robertparkinson...isn't this his username?

Reply:
slack got rid of usernames, they have display names which are slightly different, old usernames sometimes work, docs on getting the member id are here:
https://github.com/hmcts/github-slack-user-mappings#github---slack-usernames

Scroll down in the mapping file to see other examples